torch/_functorch/autograd_function.py

```diff
@@ -302,22 +302,32 @@ def custom_function_call_vmap(interpreter, autograd_function, *operands):
             f"https://pytorch.org/docs/main/notes/extending.func.html"
         )
 
+    return custom_function_call_vmap_helper(
+        interpreter, autograd_function.vmap, autograd_function, *operands, **kwargs
+    )
```

Comment:
This is for the autograd.Function case, right? Do we test somewhere that the kwargs are non-Tensors?

Reply:
I added a schema check in `register_vmap` to check that the kwargs are non-Tensors. Do we need to test it anywhere else?

Reply:
Let's add a test for an autograd.Function with vmap staticmethod and that has Tensor in the kwarg-only args.

I'm not sure that kwargs-only Tensors is actually supported in autograd.Function, would be good to check that it raises a sane error message